lib/beetle/message.rb

```diff
@@ -46,12 +46,16 @@ class Message
     attr_reader :timeout
     # how long to wait before retrying the message handler
     attr_reader :delay
+    # how long to wait before even starting the message handler
+    attr_reader :defer
     # maximum wait time for message handler retries (uses exponential backoff)
     attr_reader :max_delay
     # how many times we should try to run the handler
     attr_reader :attempts_limit
     # how many exceptions we should tolerate before giving up
     attr_reader :exceptions_limit
+    # array of exceptions accepted to be rescued and retried
+    attr_reader :on_exceptions
```

Comment:
a nice feature anyway although not necessary in order to delay/defer a message handling